### PR TITLE
Contested Claims | Attorney Fees | Migration to add type and notes to claimant table for unlisted claimants

### PR DESCRIPTION
**Pull request by leikkisa**
connects #14380

### Description
This PR is a migration to add the "type" and "notes" field for unlisted claimants where we do not have other data, and initially populates the type column with "Claimant".

### Acceptance Criteria
- [ ] Code compiles correctly

### Database Changes
*Only for Schema Changes*

* [ ] Timestamps (created_at, updated_at) for new tables
* [x] Column comments updated
* [ ] Query profiling performed (eyeball Rails log, check bullet and fasterer output)
* [ ] Appropriate indexes added (especially for foreign keys, polymorphic columns, and unique constraints)
* [x] DB schema docs updated with `make docs`
* [x] #appeals-schema notified with summary and link to this PR

### CODE REVIEW

**Comment by leikkisa** [comment]
I got some warnings when I ran "Make docs" and it looks like there's some extra deletions in the PR.  Also, should I add an "index" to the "type" column if the data isn't going to be very non-unique?

File: docs/schema/etl.csv [hunk 1]
```diff
@@ -108,103 +108,6 @@ decision_issues,rating_issue_reference_id,integer (8),,,,,x,decision_issues.rati
 decision_issues,rating_profile_date,datetime,,,,,,decision_issues.rating_profile_date
 decision_issues,rating_promulgation_date,datetime,,,,,,decision_issues.rating_promulgation_date
 decision_issues,updated_at,datetime ∗,x,,,,x,Default created_at/updated_at for the ETL record
-decision_reviews,,,,,,,,Decision Reviews
```
Comment (leikkisa) [inline docs/schema/etl.csv:111]: Not sure what all of these changes are that are unrelated to claimants.

File: db/migrate/20200608165840_add_claimant_type_and_notes.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class AddClaimantTypeAndNotes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :claimants, :type, :string, comment: "Type of claimant, such as veteran, dependent, attorney or other."
```
Comment (nanotone) [inline db/migrate/20200608165840_add_claimant_type_and_notes.rb:3]: We don't do a great job of documenting these, but I think this comment should mention that the type is a model class name, or otherwise reference STI.

File: docs/schema/etl.csv (hunk 1 above)
Comment (pkarman) [reply docs/schema/etl.csv:111]: @leikkisa you likely haven't run migrations on your local ETL db in a while.

Try `make migrate` to get all migrations run.

File: db/migrate/20200608165840_add_claimant_type_and_notes.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class AddClaimantTypeAndNotes < ActiveRecord::Migration[5.2]
+  def up
+    add_column :claimants, :type, :string, comment: "The class name for the single table inheritance type of Claimant, for example Veteran, Dependent, Attorney, or Other."
```
Comment (nanotone) [inline db/migrate/20200608165840_add_claimant_type_and_notes.rb:3]: I think we'll probably want to suffix all subclass names with `Claimant`, for the sake of clarity and consistency. And at the very least, `Veteran` is already the name of another class.
